recipes/holoviews-1.5.0/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: holoviews
+  version: 1.5.0
+
+source:
+  path: .
+
+requirements:
+  build:
+    - python
+    - param
+    - numpy
```

Comment:
Do you know if holoviews depends on numpy's ABI? (Look if numpy is import at build time and if `numpy.include` is used inside `setup.py`.)